src/OpenLoco/Widget.cpp

```diff
@@ -274,7 +275,7 @@ namespace OpenLoco::Ui
             assert(false);
         }
 
-        if (!isColourSet && imageId.hasPrimary())
+        if (!isColourSet)
```

Comment:
i made this same change in 3 places here, `drawButtonWithColour` and `drawTab` are we sure those ones aren't affected as well.